Source/ChocolateyGui/ViewModels/SourcesViewModel.cs

```diff
@@ -90,7 +90,7 @@ protected override void OnViewReady(object view)
         {
             Observable.FromEventPattern<PropertyChangedEventArgs>(this, nameof(PropertyChanged))
                 .Where(p => p.EventArgs.PropertyName == nameof(ActiveItem))
-                .Subscribe(p => DisplayName = $"Source - {ActiveItem.DisplayName}");
+                .Subscribe(p => DisplayName = $"Source - {ActiveItem?.DisplayName}");
```

Comment:
Go ahead and keep it. I'll fix it up in my other PR, as that's looking like it won't get merged for a few days yet still.